docs/readthedocs/usersguide.md

```diff
@@ -1,224 +1,213 @@
+
+![](../readthedocs/images/usersguide/copy-path.png)
+
+<li> Go to the window "Select Run/Debug Configuration" and select "Edit Configurations" </li>
 
-#. 
-   Find the ``vn_simona.conf`` file in your project directory and copy its path from the content root
+![](../readthedocs/images/usersguide/edit-conf.png)
 
-   .. figure:: ./images/usersguide/copy-path.png
+<li> Enter <code>--config={path from the content root of config file}</code> </li>
 
+> **_NOTE:_**
+For our example that would be ``--config=inputData/vn_simona/vn_simona.conf``
 
-#. 
-   Go to the window "Select Run/Debug Configuration" and select "Edit Configurations"
+![](../readthedocs/images/usersguide/edit-conf2.png)
 
-    
-   .. figure:: ./images/usersguide/edit-conf.png
+<li> Click Apply on the bottom right </li>
+</ol>
 
 
-#. 
-   Enter ``--config={path from the content root of config file}``
-   
-   .. note::
-    For our example that would be ``--config=inputData/vn_simona/vn_simona.conf``
-   
-   .. figure:: ./images/usersguide/edit-conf2.png
+### Configuring your Simulation
 
-#. 
-   Click Apply on the bottom right  
 
-Configuring your Simulation
--------------------------------
+If you want to change specific parameters of the simulation, you can create a custom configuration file.
+For a detailed description on how to do that and all the various configuration parameters you can adjust, see /config/config.
 
-If you want to change specific parameters of the simulation, you can create a custom configuration file. 
-For a detailed description on how to do that and all the various configuration parameters you can adjust, see :doc:`/config/config`.
 
+### Model and Grid Data
 
-Model and Grid Data
--------------------
 
 Besides a simulation configuration, the specifications of each grid component (e.g. lines, transformers, ...) and system participant (e.g. loads, pv plants, ... ) have to be fed into the simulation.
 Within SIMONA we use the PowerSystemDataModel (PSDM) for modeling the system components.
 Before the data can be utilized for a simulation run, make sure to convert them to the PSDM.
-For more information on the PSDM visit the `docs <https://powersystemdatamodel.readthedocs.io/en/latest/index.html>`_ and for an example of how the converted data looks like you can take a look at an example grid at ``./inputData/vn_simona/fullGrid``.
+For more information on the PSDM visit the [docs](https://powersystemdatamodel.readthedocs.io/en/latest/index.html) and for an example of how the converted data looks like you can take a look at an example grid at ``./inputData/vn_simona/fullGrid``.
 The example grids are provided as csv files. You can choose to use a different data source for your own grid.
-For more information on supported sources, check out the :doc:`/config/config` section.
+For more information on supported sources, check out the /config/config section.
```

Comment:
This link doesn't work (it didn't work before either), but since I can't figure out where it should link to, this is fine for now.

Reply:
Added new issue:  #224